sstables/sstables.cc

```diff
@@ -1810,6 +1810,15 @@ sstable::read_scylla_metadata() noexcept {
     });
 }
 
+static sstable_column_kind to_sstable_column_kind(column_kind k) {
+    switch (k) {
+        case column_kind::partition_key: return sstable_column_kind::partition_key;
+        case column_kind::clustering_key: return sstable_column_kind::clustering_key;
+        case column_kind::static_column: return sstable_column_kind::static_column;
+        case column_kind::regular_column: return sstable_column_kind::regular_column;
+    }
+}
```

Comment:
This will error on gcc, since it's legal to have an enum outside the named values. You can on_internal_error here.